static/src/js/components/GranuleFilters/__tests__/GranuleFiltersForm.test.js

```diff
@@ -907,17 +946,21 @@ describe('GranuleFiltersForm component', () => {
           },
           values: {
             equatorCrossingDate: {
-              startDate: '2019-08-14T00:00:00:000Z'
+              startDate: '2019-06-01 00:00:00'
```

Comment:
Old timing was resulting in console warning `  console.warn
    Deprecation warning: value provided is not in a recognized RFC2822 or ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non RFC2822/ISO date formats are discouraged. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.
    Arguments:
    [0] _isAMomentObject: true, _isUTC: false, _useUTC: false, _l: undefined, _i: 2019-08-14T00:00:00:000Z, _f: undefined, _strict: undefined, _locale: [object Object]
    Error:
        at Function.createFromInputFallback (/Users/eaolivar/git/earthdata-search/node_modules/moment/moment.js:324:25)`

Reply:
Put these back with orignial time look into why warning is happening